src/main/webapp/js/utils/form/form-utils.jsx

```diff
@@ -115,15 +115,22 @@ const renderFormField = (props) => {
   );
 };
 
-export const validate = fieldConfig => (values) => {
+export const validate = (fieldConfig, entities) => (values) => {
   const errors = {};
-
   _.forEach(fieldConfig, (config) => {
     const val = values[config.name];
 
     if (config.required && (_.isNil(val) || val === '')) {
       errors[config.name] = `${config.displayName} is required`;
     }
+    if (config.uniqueField && !(_.isNil(val) || val === '')) {
+      const isUniqueField = _.find(
```

Comment:
the name of this conts is misleading, this will be an entity with the same value in the unique field, so it should be named appropriately